com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/HDRP/Water/ComputeWaterVertexPositionNode.cs

```diff
@@ -0,0 +1,38 @@
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+    internal class ComputeWaterVertexPositionNode : IStandardNode
+    {
+        public static string Name => "ComputeWaterVertexPosition";
+        public static int Version => 1;
+
+        public static FunctionDescriptor FunctionDescriptor => new(
+            Name,
+@"Out = GetWaterVertexPosition(positionWS);",
+            new ParameterDescriptor[]
+            {
+                new ParameterDescriptor("positionWS", TYPE.Vec3, Usage.Local, REF.WorldSpace_Position),
+                new ParameterDescriptor("Out", TYPE.Vec3, Usage.Out)
```

Comment:
Do you want to name this PositionWS as in SG1 or was this a deliberate change?

Reply:
yes I changed it to "out" in purpose, I feel the original name is a bit duplicate since the node only has one output and the name of the node is "vertex position". What do you think?